### PR TITLE
XRT-1956: Added Jenkinsfile

### PR DESCRIPTION
**Pull request by AIM289**

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+. ./scripts/env.sh
+
+ARCHIVE=$1
+
+docker build --no-cache --tag $EXP/zigbee2mqtt:$VER-$ARCH --build-arg BUILD_VER=$VER --build-arg BUILD_SHA=$SHA --file ./docker/Dockerfile .
+
+#if [ "$ARCHIVE" = true ]; then
```
Comment (FraserG01) [inline scripts/build.sh]: need to add the docker push here

File: scripts/env.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+fi
+
+case $MNAME in
+  aarch64)
+    ARCH=arm64
+    ;;
+  armv7l)
+    ARCH=arm
+    ;;
+  i686)
```
Comment (FraserG01) [inline scripts/env.sh]: we cant support x86 with the base docker image used so just remove this case

File: VERSION [hunk 3]
```diff
@@ -0,0 +1 @@
+0.0.1.dev
```
Comment (FraserG01) [inline VERSION]: need to discuss versioning

**Review by FraserG01 — CHANGES_REQUESTED** [review]

File: scripts/build.sh (hunk 1 above)
Comment (AIM289) [reply scripts/build.sh]: added

**Review by FraserG01 — APPROVED** [review]
LGTM

File: VERSION (hunk 3 above)
Comment (AIM289) [reply VERSION]: set to 1.27.2. Not fully up to date with original Z2MQTT repo (now on 1.28.2). Once we merge 1.27 branch into master later I will then update master from original repo.

**Pull request merged**